components/server/src/auth/resource-access.ts

```diff
@@ -88,6 +89,11 @@ export interface GuardedTeam {
     members: TeamMemberInfo[];
 }
 
+export interface GuardedProject {
+    kind: "project";
+    subject: Project;
+}
```

Comment:
**Nit**: I think the guarded resource is always the team. Please remove this for now (especially since it isn't used anywhere).

Reply:
removing...